src/backfill/backfill.types.ts

```diff
@@ -1,6 +1,6 @@
 import { RepoSyncState } from "models/reposyncstate";
 
-export type JobId = {
+export interface JobId {
```

Comment:
Hi @atraversatlassian , do you know why's the recommendation of interface over type?

Reply:
This was changed from this rule https://typescript-eslint.io/rules/consistent-type-definitions/

Reply:
Thanks. From the link I see the reason is "Keep declaration consistent", but not mentioning why `interface` is better than `type`. So I guess the rule is to help being consistent and sort of pick `interface`?

Reply:
And this is me out of curious, I am fine with changing from type to interface.